l10n_vn_account_invoice_number/models/account_invoice.py

```diff
@@ -0,0 +1,50 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Trobz (http://trobz.com)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models, _
+from odoo.exceptions import UserError
+
+
+class AccountInvoice(models.Model):
+    _inherit = "account.invoice"
+
+    tax_invoice_number = fields.Integer()
+    tax_invoice_form = fields.Char()
+    tax_invoice_serie = fields.Char()
+    is_owner_in_vietnam = fields.Boolean(string='Is Owner in Vietnam',
+                                         store=False,
+                                         compute='_compute_is_owner_in_vietnam')
```

Comment:
======== Testing test_flake8 ========
l10n_vn_account_invoice_number/models/account_invoice.py:17:80: E501 line too long (80 > 79 characters)

Need to fix :smiley:

Reply:
Crazy bot :(